scripts/wsrep_sst_common.sh

```diff
@@ -430,6 +528,41 @@ function check_for_version()
 #   0 if successful (no errors encountered)
 #   <non-zero> otherwise
 #
+# Overview: There are two code paths:
+#   (1) Receiving from a < 8.0 donor (5.7 donor)
+#   (2) Receiving from a >= 8.0 donor
+#
+#   (1) Receiving from a 5.7 donor
```

Comment:
Can you expand this comment to make it more explicit that 8.0 binaries will not allow CREATE USER TO work on DONOR copied 5.7 data-directory.

Reply:
There's a comment below (with the detailed flow) that explains this.